### DISCUSSION TITLE
Temp Check | DCDAO 2024 Q4 Budget Application

### DISCUSSION THREAD

**Discussion by furoxr** [ringecosystem/discussion#11] [General]
**Date**: 21/11/2024

**Requested Amount**: 17,864,806.87 RING

**Receiving Address**: 0x8683A0ED2bAdb1F3AaBeA6686d78649F0Da774A0

## 1. Applicant Identity

The Darwinia Community DAO (DCDAO) is a decentralized, member-governed entity dedicated to supporting the Darwinia ecosystem through community engagement, marketing, development support, and transparent governance.

## 2. About DCDAO

**DCDAO Overview**:

- A community self-organized workgroup (WG) aiming to contribute to the growth and development of the Darwinia community.
- Focus areas:
  - **Community Engagement and Support**: Enhance communication, provide educational resources, and assist community members.
  - **Marketing and Promotion**: Create awareness through social media, events, and collaborations.
  - **Development Support**: Facilitate the creation of tools, applications, and infrastructure for Darwinia.
  - **Governance and Decision-Making**: Encourage community participation and transparency.

[More information](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x257aff1540719475411091706f7b200507fe9b9fe7397a8a4347fbc9d7234cb6)

## 3. Budget Details (Q4 2024)

Note: Any surplus carries over to the next quarter if not used up by DCDAO.

| Program                                           | Monthly Requested Amount (USD) | Quarterly Requested Amount (USD) | Total (RING) | RING Price |
| ------------------------------------------------- | ------------------------------ | -------------------------------- | ------------ | ---------- |
| Community Growth Program                          | 1,500.00                       | 4,500.00                         | 2,414,163.09 | 0.001864   |
| Management, Maintenance, and Development Proposal | 3,500.00                       | 10,500.00                        | 5,633,047.21 | 0.001864   |
| Community Manager Proposal                        | 5,000.00                       | 15,000.00                        | 8,047,210.30 | 0.001864   |
| Treasury Specialist                               | 600.00                         | 1,800.00                         | 965,665.24   | 0.001864   |
| Steering Committee Compensation                   | 500.00                         | 1,500.00                         | 804,721.03   | 0.001864   |

&#x20;

Budget Adjustments:

- Steering Committee Payment: Reduced to 500 USDT per month for two bi-weekly meetings (previously 1,000 USDT for four weekly meetings).
- Treasury Specialist Payment: Reduced to 600 USDT per month (previously 800 USDT).
- Community Growth Program: Adjusted to 1,500 USDT.
- Marketing and Branding Program: Adjusted to 3,500 USDT, Infura Program canceled.

These changes are  parts of DCDAO’s efforts to optimize operational costs in response to bear market pressures and ensure the longevity of the project.

### 4.1 Community

- Managed DAO staking and progressed discussions around KTON/RING LP Farm, treasury voting delegations, and liquidity sources.
- Promoted an Advocate to Ambassador and supported the RingDAO Marketing Action plan.
- Boosted community engagement with weekly quizzes (20-25+ participants), Twitter updates, monthly articles, and GitHub discussions.
- Supported KtonDAO, RingDAO, Helix Discord, and kept community channels spam-free.

### 4.2 Marketing and Branding Management, Maintenance, and Developer Education

- Marketing and Branding: Developed and executed marketing strategies, established a marketing board for coordination, and proposed a marketing strategy for RingDAO.
- Community Engagement: Represented Darwinia at events such as Token2049, Polkadot Decoded, and Cosmoverse 2024, securing a speaking opportunity and reinforcing community ties.
- Developer Education: Created and published multiple videos for the Darwinia Developer Series, focusing on topics like token transfers and smart contract deployment.
- Website Maintenance: Maintained the Darwinia.Network website and launched the RingDAO site, ensuring they were up-to-date and functional.
- Paralink and Vote.RingDAO.Com: Completed the development of Paralink and made significant updates to the Vote.RingDAO.Com platform, including UI enhancements and support for additional networks.

### 4.3 Growth

- Hosted multiple DCDAO Community Calls to engage the community.
- Expanded the Darwinia Partner Network, adding Tashi Protocol and Composable Security.
- Promoted $RING through targeted marketing campaigns, including articles and outreach.
- Supported ETHCC and Polkadot Decoded events to advance ecosystem goals.
- Organized the CKTON/CRAB liquidity event to enhance token utility.
- Conducted Spanish-language outreach to expand community engagement.

## 5. Governance and Constitution

- [Transition To Tally](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xe767cfef64633039225d935daaa24f5050b7e3c267efa74c447a3a25a982d467)
- [DCDAO Constitution](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xd750140eccee3434f65feb156c985f6425ed884d56aa2a295c189d5e76fe9dc5)
- [Super Majority Voting Requirement](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xf54f569ed893d5391b7b4dc716aa8b52bc99c2ac170c8beb43f98cb1b4ce7557)
- [Member Join Proposal](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xadf0533b8e283bbfacb61e69c83f6ca3b624d4de0c78a0a29883ebe6adc85bcb)
- [Steering Committee Proposal](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x2e17e91c48afaf8233e5b9fed3ea48cc3b572a7f36d895732619fd64d5fa5331)

## 6. Team

**Managers**:

- Community Manager: Jesse
- Community Growth Manager: David
- Management, Maintenance, and Development: Furqan

**Steering Committee**:

- Denny
- Aki
- Furqan
- Eve
- Nada

**Ambassadors**:

- Laki
- Saso
- June
- John

**Advocates**:

- Lephofora
- Daniel

## 7. DCDAO Treasury

**Treasury Addresses**:

- [Ethereum](https://etherscan.io/address/0x195741BC761B25A89fc7798087652f97f29B306a)
- [Arbitrum](https://arbiscan.io/address/0x0DF9F845D33a49e259A0A20FD814bcf7e9BA6048)
- [Polygon](https://polygonscan.com/address/0x28760448c28617047BCB2a756e2442567c23ED81)
- [Darwinia](https://darwinia.subscan.io/address/0x8683A0ED2bAdb1F3AaBeA6686d78649F0Da774A0)
- [Optimism](https://optimistic.etherscan.io/address/0x26102b33bB985bdD6a46D50A692338F0881ff5Cf)

*Note*: Multisig accounts are used only for executing decisions made explicitly in proposals. They reserve the right to refuse execution if the proposal lacks clarity.

**Comment by hackfisher** [comment]
Due to the successful progress of the budget adjustment plan, as a RingDAO Delegate, I am inclined to support this proposal. However, the quarterly budget still accounts for approximately 5% of the Treasury, which remains a significant amount. Therefore, I recommend that DCDAO continue implementing reforms to reduce its dependency on the Treasury by encouraging budget reductions or diversifying funding sources. For example:

1. Encourage volunteer contributions from RING holders: As a community DAO, DCDAO should prioritize volunteer efforts from RING holders rather than relying primarily on compensation and salaries.
	
2. Focus on incubation and fostering connections: As RingDAO’s community organization, DCDAO should aim to serve more as a platform for incubation and fostering collaborations. For later-stage projects, it should guide the involvement of external resources and funding. Relying solely on the RingDAO Treasury for incubation could lead to mismatches and sustainability challenges.

**Comment by 2xJDubs** [comment]
I am in support of this proposal.  I also agree with hackfisher on needing continued efforts to find new sources of funding.  For Q1 I will be reducing the Community Manager funding to better align with other programs and reduce reliance on the treasury.

**Reply by 2xJDubs** [reply]
To add to this, I am okay reducing the Q4 budget for Community Manager to $4,000 USDT per month.  Part of my Q4 proposal for CM  included $1K per quarter for community rewards like what we give away for the quiz games each week but myself and the other Ambassadors are happy to keep moving this forward personally.  I will be removing the community engagement funding part of the Q1 proposal when I post it.

**Comment by hackfisher** [comment]
@furoxr 

To proceed on Tally, please request @hujw77 to provide the action parameters for proposal.

@hujw77 To use custom action for transfer RING from RingDAO treasury to target address, we need an intermediate contract helping forward.

**Comment by hujw77** [comment]
### Proposal Action Params
```
Target: 0x1886AaB802B13C4d96185bd2E18e0Fe60FBC7993
Function: transfer(address to)
  - to: <The RING receive address>
Value: <The requested amount of RING>
```
![Screenshot 2024-12-04 at 16 12 32](https://github.com/user-attachments/assets/0972e6d0-553b-4662-a9a5-5145b7694278)
![Screenshot 2024-12-04 at 16 14 34](https://github.com/user-attachments/assets/7a68decf-be0d-4441-a2e3-2ac99d557a7e)

**Comment by hackfisher** [comment]
https://darwinia.subscan.io/block/4873847?tab=event&event=4873847-1  @furoxr

**Reply by hackfisher** [reply]
https://explorer.darwinia.network/address/0x4DCf0f14cC58fc2Bf313e70573dDB7309523bb23

**Comment by furoxr** [comment]
https://www.tally.xyz/gov/ringdao/proposal/39167772932143723025658918622015993797875961063645251007065064214127031243174
